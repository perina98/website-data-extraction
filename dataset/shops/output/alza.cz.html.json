[
    {
        "sale": "-2%",
        "name": "12 390,-",
        "price": "12 190,-"
    },
    {
        "sale": "-16%",
        "name": "12 990,-",
        "price": "10 899,-"
    },
    {
        "sale": "-2%",
        "name": "19 990,-",
        "price": "19 590,-"
    },
    {
        "sale": "-8%",
        "name": "12 590,-",
        "price": "11 590,-"
    },
    {
        "sale": "-20%",
        "name": "14 990,-",
        "price": "11 990,-"
    },
    {
        "sale": "-13%",
        "name": "10 390,-",
        "price": "8 999,-"
    }
]