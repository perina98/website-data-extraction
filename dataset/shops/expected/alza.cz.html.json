[
    {
        "name": "AMD RYZEN 5 3600",
        "description": "Procesor 6 jádrový, 12 vláken, 3,6GHz (TDP 65W), Boost 4,2 GHz, 32MB L3 cache, socket AMD AM4, Matisse, box chladič, Wraith Stealth",
        "price": "5 090,-",
        "sale": "-6%"
    },
    {
        "name": "Intel Core i7-9700F",
        "description": "Procesor 8 jádrový, 8 vláken, 3GHz (TDP 65W), Turboboost 4,7GHz, 12MB L3 cache, socket Intel 1151, Coffee Lake refresh, box chladič, pouze chipset Intel řady 3xx",
        "price": "5 799,-",
        "sale": "-3%"
    },
    {
        "name": "AMD Ryzen 7 5800X",
        "description": "Procesor 8 jádrový, 16 vláken, 3,8GHz (TDP 105W), Boost 4,7 GHz, 32MB L3 cache, Bez integrovaného grafického čipu, socket AMD AM4, Vermeer, bez chladiče",
        "price": "12 190,-",
        "sale": "-2%"
    },
    {
        "name": "AMD RYZEN 5 1600 (12nm)",
        "description": "Procesor 6 jádrový, 12 vláken, 3,2GHz (TDP 65W), Boost 3,6 GHz, 16MB L3 cache, socket AMD AM4, Pinnacle Ridge, box chladič, Wraith Stealth",
        "price": "3 290,-"
    },
    {
        "name": "Intel Core i5-10400F",
        "description": "Procesor 6 jádrový, 12 vláken, 2,9GHz (TDP 65W), Boost 4,3 GHz, 12MB L3 cache, Bez integrovaného grafického čipu, socket Intel 1200, Comet Lake, box chladič",
        "price": "3 799,-",
        "sale": "-3%"
    },
    {
        "name": "Intel Core i5-8600",
        "description": "Procesor 6 jádrový, 6 vláken, 3,1GHz (TDP 65W), Turboboost 4,3GHz, 9MB L3 cache, Intel UHD Graphics 630 350MHz, socket Intel 1151, Coffee Lake, box chladič, pouze chipset Intel řady 3xx",
        "price": "4 699,-"
    },
    {
        "name": "Intel Core i5-10600KF",
        "description": "Procesor 6 jádrový, 12 vláken, 4,1GHz (TDP 125W), Boost 4,8 GHz, 12MB L3 cache, Bez integrovaného grafického čipu, socket Intel 1200, Comet Lake, bez chladiče",
        "price": "5 299,-",
        "sale": "-15%"
    },
    {
        "name": "AMD RYZEN 5 3500X",
        "description": "Procesor 6 jádrový, 6 vláken, 3,6GHz (TDP 65W), Boost 4,1 GHz, 32MB L3 cache, Bez integrovaného grafického čipu, socket AMD AM4, Matisse, box chladič, Wraith Stealth",
        "price": "4 290,-"
    },
    {
        "name": "AMD RYZEN 7 3700X",
        "description": "Procesor 8 jádrový, 16 vláken, 3,6GHz (TDP 65W), Boost 4,4 GHz, 32MB L3 cache, socket AMD AM4, Matisse, box chladič, Wraith Prism RGB",
        "price": "8 490,-"
    },
    {
        "name": "Intel Core i9-10850K",
        "description": "Procesor 10 jádrový, 20 vláken, 3,6GHz (TDP 125W), Boost 5,2 GHz, 20MB L3 cache, Intel UHD Graphics 630 1,2GHz, socket Intel 1200, Comet Lake, bez chladiče, pouze chipset Intel řady 4xx",
        "price": "10 899,-",
        "sale": "-16%"
    },
    {
        "name": "Intel Core i5-9400F",
        "description": "Procesor 6 jádrový, 6 vláken, 2,9GHz (TDP 65W), Turboboost 4,1GHz, 9MB L3 cache, socket Intel 1151, Coffee Lake refresh, box chladič, pouze chipset Intel řady 3xx",
        "price": "3 299,-",
        "sale": "-3%"
    },
    {
        "name": "AMD RYZEN 9 3950X",
        "description": "Procesor 16 jádrový, 32 vláken, 3,5GHz (TDP 105W), Boost 4,7 GHz, 64MB L3 cache, socket AMD AM4, Matisse, bez chladiče",
        "price": "19 590,-",
        "sale": "-2%"
    },
    {
        "name": "Intel Core i5-10400",
        "description": "Procesor 6 jádrový, 12 vláken, 2,9GHz (TDP 65W), Boost 4,3 GHz, 12MB L3 cache, Intel UHD Graphics 630 1100MHz, socket Intel 1200, Comet Lake, box chladič",
        "price": "4 549,-"
    },
    {
        "name": "AMD RYZEN 9 3900X",
        "description": "Procesor 12 jádrový, 24 vláken, 3,8GHz (TDP 105W), Boost 4,6 GHz, 64MB L3 cache, socket AMD AM4, Matisse, box chladič, Wraith Prism RGB",
        "price": "11 590,-",
        "sale": "-8%"
    },
    {
        "name": "Intel Core i9-10900K",
        "description": "Procesor 10 jádrový, 20 vláken, 3,7GHz (TDP 125W), Boost 5,2 GHz, 20MB L3 cache, Intel UHD Graphics 630 1200MHz, socket Intel 1200, Comet Lake, bez chladiče, pouze chipset Intel řady 4xx",
        "price": "11 990,-",
        "sale": "-20%"
    },
    {
        "name": "AMD RYZEN 5 2600",
        "description": "Procesor 6 jádrový, 12 vláken, 3,4GHz (TDP 65W), Boost 3,9 GHz, 16MB L3 cache, socket AMD AM4, Pinnacle Ridge, box chladič, Wraith Stealth",
        "price": "4 290,-"
    },
    {
        "name": "Intel Celeron G5920",
        "description": "Procesor 2 jádrový, 2 vlákna, 3,5GHz (TDP 58W), 2MB L3 cache, Intel UHD Graphics 610 350MHz, socket Intel 1200, Comet Lake, box chladič, pouze chipset Intel řady 4xx",
        "price": "1 099,-"
    },
    {
        "name": "Intel Core i7-10700KF",
        "description": "Procesor 8 jádrový, 16 vláken, 3,8GHz (TDP 125W), Boost 5,1 GHz, 16MB L3 cache, Bez integrovaného grafického čipu, socket Intel 1200, Comet Lake, bez chladiče",
        "price": "8 399,-",
        "sale": "-13%"
    },
    {
        "name": "AMD RYZEN 7 3800X",
        "description": "Procesor 8 jádrový, 16 vláken, 3,9GHz (TDP 105W), Boost 4,5 GHz, 32MB L3 cache, socket AMD AM4, Matisse, box chladič, Wraith Prism RGB",
        "price": "8 990,-",
        "sale": "-5%"
    },
    {
        "name": "Intel Core i3-10100F",
        "description": "Procesor 4 jádrový, 8 vláken, 3,6GHz (TDP 65W), Boost 4,3 GHz, 6MB L3 cache, Bez integrovaného grafického čipu, socket Intel 1200, Comet Lake, box chladič, pouze chipset Intel řady 4xx",
        "price": "2 299,-"
    },
    {
        "name": "Intel Core i7-10700K",
        "description": "Procesor 8 jádrový, 16 vláken, 3,8GHz (TDP 125W), Boost 5,1 GHz, 16MB L3 cache, Intel UHD Graphics 630 1200MHz, socket Intel 1200, Comet Lake, bez chladiče",
        "price": "8 999,-",
        "sale": "-13%"
    },
    {
        "name": "Intel Core i9-9900K",
        "description": "Procesor 8 jádrový, 16 vláken, 3,6GHz (TDP 95W), Turboboost 5GHz, 16MB L3 cache, Intel UHD Graphics 630 1200MHz, socket Intel 1151, Coffee Lake refresh, bez chladiče, pouze chipset Intel řady 3xx",
        "price": "9 499,-"
    },
    {
        "name": "Intel Core i7-10700",
        "description": "Procesor 8 jádrový, 16 vláken, 2,9GHz (TDP 65W), Boost 4,8 GHz, 16MB L3 cache, Intel UHD Graphics 630 1200MHz, socket Intel 1200, Comet Lake, box chladič",
        "price": "7 699,-",
        "sale": "-14%"
    },
    {
        "name": "Intel Pentium G6400",
        "description": "Procesor 2 jádrový, 4 vlákna, 4GHz (TDP 58W), 4MB L3 cache, Intel UHD Graphics 610 350MHz, socket Intel 1200, Comet Lake, box chladič, pouze chipset Intel řady 4xx",
        "price": "1 599,-"
    }
]